### ISSUE TITLE
Folders created in Revo using the manager interface are created with permission 041

### ISSUE THREAD

**Issue by xpix** [modxbot/migrate#10753]
xpix created Redmine issue ID 10753

Folders created in Revo using the manager interface are created with permission 041. I have to use the command line to change the permission to 755 Thank you

**Comment by splittingred** [comment]
splittingred submitted:

This seems to be more a problem with your setup - do you happen to have the new_folder_permissions setting set? Otherwise your umask is not returning the right value, which is a PHP config problem.

**Comment by xpix** [comment]
xpix submitted:

I can't find the new_folder_permissions option in the config for rev6883. I tried filtering by new, folder or perm but no results. I might be missing something. Thanks

**Comment by splittingred** [comment]
splittingred submitted:

Add a Setting called 'new_folder_permissions', and set it to 0755. Then try and create a directory.

**Comment by pixelchutes** [comment]
pixelchutes submitted:

I too was wondering where "new_folder_permissions" was. What is the setting key for new files, "new_file_permissions"? I was looking for a way to change the default chmod value set during installation, but could not locate a way to change this post-install...

**Comment by opengeek** [comment]
opengeek submitted:

It's not set unless you modify the permissions from whatever the server's umask() is. I'm not sure this is the best way to do it, but I wanted to respect the umask unless you explicitly override it.

**Comment by opengeek** [comment]
opengeek submitted:

Is this still happening for you cipa? If so, can you provide some information on the server umask, the user account php is running as/how it's running, and any other relevant platform details?

**Comment by xpix** [comment]
xpix submitted:

This still happens for rev6987. I created a folder using the "Insert Image" functionality for TinyMce and the folder was created with 041 permission. The owner and group are correct. I also created a folder through Cyberduck and the permission is correctly set to 755. Server info:
PHP Version 5.2.12
MySql 5.0.90-community Unfortunately I have no idea how to find the server umask. Maybe you know a command I can run in the terminal. PS: I'll add new_folder_permissions to the config setting and post the results here
